### ISSUE TITLE
[NFR] Server templates or saving

### ISSUE THREAD

**Issue by uLow** [REBELinBLUE/deployer#344]
**Problem:**
When you have one server to deploy many projects, it's very annoying to copy-paste whole server configuration every time.

**Solution:**
Allow to choose server settings from templates or already used servers. In fact, all we need to change in such situation is `project_path`.

I guess this could be done within one business day. @REBELinBLUE, would you accept that challenge? :)

**Comment by REBELinBLUE** [comment]
Someone has actually done a PR for server "templates" which is now in master but there are some changes I want to make to it before I release it, just a bit swamped with other stuff at the moment :|

**Comment by uLow** [comment]
Yeah, found them. But can't figure out idea about not including user in that template and probably default part of path.

What are you going to change there?

**Comment by REBELinBLUE** [comment]
Sorry, been very busy the last few months. I am completely rewriting it as I don't like the way it is done. "Shared servers" will be just that, if you change a setting it will apply to all projects rather than new ones as with the "server templates" that are currently there. Shared servers will have all the usual details, name, address, port, username and path. Username and path will be possible to overwrite on a per project basis
